client/trackercyf/src/mentor.js

```diff
@@ -0,0 +1,69 @@
+// User stories we use when creating code.
+// As a mentor, I want to see the list of the homework
+// As a mentor, I want to add modules with tasks to the list of homework
+
+// version 1. this code is a duplicate of the version1 in trainee.js.
+// It will be necessary to add permission to add and delete modules and tasks. this version has tickbox, modules, tasks, submit button
+import React, { useState } from "react";
+
+function Mentor() {
+  // State variables for the TaskTracker component
+  const [taskCompleted, setTaskCompleted] = useState(false);
+  const [feedback, setFeedback] = useState("");
```

Comment:
why would mentor complete or not not complete task? i was imagining a mentor only adds tasks and can check who as completed, and same with the feedback

Reply:
Agree. I update next versions